common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -1184,6 +1185,39 @@ def test_quality_button_works_correctly(self):
         self.video.wait_for(lambda: self.video.is_quality_button_active, 'waiting for quality button activation')
 
 
+@attr('shard_4')
+class DragAndDropTest(VideoBaseTest):
+    """
+    Tests draggability of closed captions within videos.
+    """
+    def setUp(self):
+        super(DragAndDropTest, self).setUp()
+
+    def test_if_captions_are_draggable(self):
+        """
+        Loads transcripts so that closed-captioning is available.
+        Ensures they are draggable.
+        """
+        self.assets.append('subs_3_yD_cEKoCk.srt.sjson')
+        data = {'sub': '3_yD_cEKoCk'}
+
+        self.metadata = self.metadata_for_mode('html5', additional_data=data)
+
+        self.navigate_to_video()
+
+        self.video.show_closed_captions()
+
+        self.video.wait_for_closed_captions()
+
+        self.assertTrue(self.video.is_closed_captions_visible)
+
+        captions_container = '.closed-captions'
+        captions = self.video.q(css=captions_container).results[0]
+
+        action = ActionChains(self.browser)
+        action.drag_and_drop_by_offset(captions, 15, 185).perform()
```

Comment:
It  seems like there should be some other assert statement in here.  A typical pattern would be something along the lines of getting the initial position of the item, performing the `drag_and_drop_by_offset`,  and then waiting for the position to be changed by the specified offset.